tests/src/test/java/docs/javadsl/SerializationTest.java

```diff
@@ -24,6 +24,12 @@
 import com.fasterxml.jackson.databind.ObjectWriter;
```

Comment:
Surprised these codegen'd files don't end up in `target`.

Reply:
I'm guessing @ennru copy/pasted them so we don't need a protobuf plugin on the `docs/` build?

Reply:
Good point